Cargo.toml

```diff
@@ -29,7 +29,7 @@ getrandom = { version = "0.2.15", default-features = false, optional = true }
 rustix = { version = "0.38.39", features = ["fs"] }
 
 [target.'cfg(windows)'.dependencies.windows-sys]
-version = ">=0.52,<=0.59"
+version = ">=0.52,<=0.60"
```

Comment:
The current version selection should work with 0.59 (the current latest version), although it'll have issues with 0.59.1 etc. This should have been `>=0.52,<0.60.0` (strictly less than).

Reply:
olol, I've been writing so many of these bump PRs that I didn't even notice the equals already there. Sigh! Good catch. 😅